src/main/java/seedu/weme/model/util/SampleDataUtil.java

```diff
@@ -40,6 +44,20 @@ public static ReadOnlyMemeBook getSampleMemeBook() {
         return sampleMb;
     }
 
+    public static StatsEngine getSampleStatsData() {
+        StatsEngine statsEngine = new StatsManager();
+        statsEngine.setLikeData(getSampleLikeData());
```

Comment:
any rationale why likedata is not just inside statsengine?

Reply:
This is just to segment out SampleLikeData. Fulfilling SRP.